builds/fabric8-che/plugins/keycloak-plugin-server/src/main/java/com/redhat/che/keycloak/server/KeycloakAuthenticationFilter.java

```diff
@@ -15,14 +15,22 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
             throws IOException, ServletException {
         HttpServletRequest request = (HttpServletRequest) req;
         String auth = request.getHeader("Authorization");
-        if(auth == null ){
+        String uri = request.getRequestURI();
+
+        if (uri.endsWith("/api/system/state")) {
```

Comment:
@amisevsk well, I deliberately moved this check to the new block, so that the code would  be more readable, otherwise it  would be lost in numerous websocket checks (IMO it would make more sense to extract those into a separate method e.g.  `isWebSocketRequest()`)